infra/cifuzz/actions/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# This uses three docker images. Bad! We only want one (or two if really needed)
+# at some point.
+FROM debian:9.5-slim
+# TODO: don't run docker in docker :-)
```

Comment:
I don't think we are using docker in docker. These are sibling containers. Old comment. Delete!

Reply:
Done.